stablehlo/reference/Ops.cpp

```diff
@@ -68,6 +81,49 @@ Tensor eval(FloorOp op, const Tensor &operand) {
   return result;
 }
 
+Tensor eval(IotaOp op) {
+  Tensor result(op.getType());
+  Type elType = result.getType().getElementType();
+  if (!isSupportedIntegerType(elType) && !isSupportedFloatType(elType) &&
+      !isSupportedComplexType(elType))
+    report_fatal_error(invalidArgument("Unsupported element type: %s",
+                                       debugString(elType).c_str()));
+  uint64_t iotaDimension = op.getIotaDimension();
+  llvm::RoundingMode rm = APFloat::rmNearestTiesToEven;
+  bool roundingErr;
```

Comment:
I think these variables should be declared next to their callsites, so that they are placed in the narrowest scope possible. Also, how much mileage are we getting from declaring a variable that carries `APFloat::rmNearestTiesToEven`? It's used only three times below, so perhaps we should just inline it?

Reply:
It was declared early to remove some clutter from the loops, but with your suggestions above, it is much more simplified, so I'll inline it.